demos/classification_demo/python/classification_demo.py

```diff
@@ -37,73 +37,89 @@
+            'The demo will look for a suitable plugin for device specified. Default is CPU')
 
     common_model_args = parser.add_argument_group('Common model options')
-    common_model_args.add_argument('--labels', help='Optional. Labels mapping file.', default=None, type=str)
-    common_model_args.add_argument('-topk', help='Optional. Number of top results. Default value is 5. Must be from 1 to 10.', default=5,
-                                   type=int, choices=range(1, 11))
-    common_model_args.add_argument('--layout', type=str, default=None,
-                                   help='Optional. Model inputs layouts. '
-                                        'Ex. NCHW or input0:NCHW,input1:NC in case of more than one input.')
+
+    common_model_args.add_argument('--labels', default=None, type=str, metavar="<LABELS>",
+        help='labels mapping file')
+
+    common_model_args.add_argument('--layout', type=str, default=None, metavar="<STRING>",
+        help='model inputs layouts. Example: NCHW or input0:NCHW,input1:NC in case of more than one input')
+
+    common_model_args.add_argument('--topk', default=5, type=int, choices=range(1, 11), metavar="<NUMBER>",
+        help='number of top results(from 1 to 10). Default value is 5')
 
     infer_args = parser.add_argument_group('Inference options')
-    infer_args.add_argument('-nireq', '--num_infer_requests', help='Optional. Number of infer requests',
-                            default=0, type=int)
-    infer_args.add_argument('-nstreams', '--num_streams',
-                            help='Optional. Number of streams to use for inference on the CPU or/and GPU in throughput '
-                                 'mode (for HETERO and MULTI device cases use format '
-                                 '<device1>:<nstreams1>,<device2>:<nstreams2> or just <nstreams>).',
-                            default='', type=str)
-    infer_args.add_argument('-nthreads', '--num_threads', default=None, type=int,
-                            help='Optional. Number of threads to use for inference on CPU (including HETERO cases).')
+
+    infer_args.add_argument('-nireq', '--num_infer_requests', default=0, type=int, metavar="<NUMBER>",
+        help='number of infer requests')
+
+    infer_args.add_argument('-nstreams', '--num_streams', default='', type=str, metavar="<NUMBER>",
+        help='number of streams to use for inference on the CPU or/and GPU in throughput mode'
+            '\n(for HETERO and MULTI device cases use format '
+            '<device1>:<nstreams1>,<device2>:<nstreams2> or just <nstreams>)')
+
+    infer_args.add_argument('-nthreads', '--num_threads', default=None, type=int, metavar="<NUMBER>",
+        help='number of threads to use for inference on CPU (including HETERO cases)')
 
     io_args = parser.add_argument_group('Input/output options')
+
+    io_args.add_argument('-lim', '--limit', required=False, default=1000, type=int, metavar="<NUMBER>",
```

Comment:
```suggestion
    io_args.add_argument('-lim', '--lim', required=False, default=1000, type=int, metavar="<NUMBER>",
```

`lim` is 3 char long, so normally we should have `--lim`, but keep `-lim` for compatibility with c++